src/main.cc

```diff
@@ -404,16 +408,14 @@ usage(void)
     exit(EXIT_FAILURE);
 }
 
-/**
- * Parse the parameters received via command line interface.
- *
- * \param argc   Number of options received on command line
- * \param argv   List of parameters received on command line
- */
-static void
-mainParseOptions(int argc, char *argv[])
-{
-    int optIndex = 0;
+/// CommandLine option IDs for --long options that lack a short (-x) equivalent
+enum {
+    // The absolute values do not matter except that the following values should
+    // not be used: Values below 2 are for special getopt_long(3) use cases, and
+    // values in the [33,126] range are reserved for short options (-x).
+    optForeground = 2,
```

Comment:
the big 'switch' in mainHandleCommandLineOption() should use this new optForeground instead of '1'.

Reply:
Please fix.

Reply:
Done.